libraries/plugins/peerplays_sidechain/sidechain_net_handler_bitcoin.cpp

```diff
@@ -1365,14 +1369,14 @@ std::string sidechain_net_handler_bitcoin::create_primary_wallet_transaction() {
          total_amount += utx.amount_;
       }
 
-      if (min_amount >= total_amount) {
+      if (fee_rate >= total_amount) {
          elog("Failed not enough BTC to transfer from ${fa}", ("fa", prev_pw_address));
          return "";
       }
    }
 
    fc::flat_map<std::string, double> outputs;
-   outputs[active_pw_address] = total_amount - min_amount;
+   outputs[active_pw_address] = double(total_amount - fee_rate) / 100000000.0;
 
```

Comment:
still working with doubles...

Reply:
This is output parameters, it doesn't really matter because of the fee,  but I'll fix it too.